imports/client/components/RelatedPuzzleGroup.tsx

```diff
@@ -37,7 +37,9 @@ function sortPuzzlesByRelevanceWithinPuzzleGroup(
 }
 
 interface RelatedPuzzleGroupProps {
-  sharedTag: TagType;
+  sharedTag: TagType | null;
+  // noSharedTagLabel is used to label the group only if sharedTag is null.
+  noSharedTagLabel: String;
```

Comment:
I was confused at first by this since we don't specify it in RelatedPuzzleGroups.tsx, but then I saw the `defaultProps` down below.  It feels a little weird, but I can roll with it, and RelatedPuzzleGroups doesn't wind up with an undefined `sharedTag` ever.

Reply:
It actually is passed through in the RelatedPuzzleGroup on PuzzleListPage (which now can support a null sharedTag). My thinking was that we use RelatedPuzzleGroup in a few contexts and the message most helpful here (that no group has even been set) wouldn’t necessarily apply to other instances of “default” groups in the future.